scanner/updater/version/RELEASE_VERSION

```diff
@@ -22,3 +22,4 @@
 4.6.0
 4.6.1
 4.6.2
+4.5.6
```

Comment:
This is a duplicate (line 4.5.6).

Reply:
@jvdm should we close this PR, then?

Reply:
@RTann yes, and then understand why it happened.

Reply:
@daynewlee you're the assignee, letting you close this so I won't be stepping on anything else you might be doing (tests, running workflows, etc).